### PR TITLE
README only re: client credentials.

### PR DESCRIPTION
**Pull request by AndyNewton**
README addition only: hoping to help folks obtain and use client credentials.
PR as offered on Slack: https://mojdt.slack.com/archives/CK0QLCQ1G/p1639646384136400

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,13 +250,36 @@ The redirect endpoint will then call:
...
+- Postman: as the value for `request | Authorization | Bearer Token | Token`
+- Swagger: where a Swagger view offers an `Authorize` control (e.g. page top-right) then as the value for
+  `Name: Authorization` `In: header ` as `Bearer <value>`
+- curl: as the value in `--header 'Authorization: Bearer <value>`
+
+A typical valid token will look like the following and be of the order 800 characters long: `"eyJhbGciOiJSUz …… QBQeyow"`
+
+#### How do I get client credentials?
 
+Raise a request in the #dps-tech-team Slack channel and then a Jira ticket will be created to
```
Comment (Mjwillis) [inline README.md:278]: The channel we will archived in the next 2 weeks as the DPT Tech team is being disbanded

Comment (Mjwillis) [reply README.md:278]: @MichelleismeMoJ will soon setup a new channel for auth support

Comment (AndyNewton) [reply README.md:278]: @Mjwillis If this update looks useful then how about merging it as is now?  
There are other mentions of "#dps-tech-team Slack channel" in the README already; they'll all need updating I presume.  Changing them all together will be easier.

**Review by Mjwillis — APPROVED** [review]
Will update dps tech globally

**Pull request merged**